1host.config.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+	port: 3000,
+  modules: [
+    {
+        module: require("./index.js"),
+			"errorHandler": false,
```

Comment:
Codacy has a fix for the issue: [Replace `"errorHandler"` with `errorHandler`](https://app.codacy.com/gh/onedotprojects/scratchCommentAuth/pullRequest?prid=7364619)
```suggestion
			errorHandler: false,
```